Guide.md

```diff
@@ -0,0 +1,287 @@
+4.  Leave _t2.micro_ selected, and click **Next: Configure Instance Details**.
+5.  On the _Configure Instance Details_ page:
+    *   _Network_: Leave default
+    *   _Subnet_: Leave default
+    *   _Auto-assign Public IP_: **Enable**
+6.  Click **Next: Add Storage** > **Next: Add Tags** > **Next: Configure Security Group**.
+7.  Click **Review and Launch**, and then **Launch**.
+8.  In the key pair dialog, select **Create a new key pair**.
+9.  Give it a _Key pair name_ of "mynvkp".
+10.  Click **Download Key Pair**, and then **Launch Instances**.
+11.  Click **View Instances**, and give it a few minutes to enter the _running_ state.
+12.  Once the instance is fully created, check the checkbox next to it and click **Connect** at the top of the window.
+13.  In the _Connect to your instance_ dialog, select **EC2 Instance Connect (browser-based SSH connection)**.
+14.  Click **Connect**.
+15.  In the command line window, check the AWS CLI version:
+
+        aws --version
+
+    It should be an older version.
+
+16.  Download v2:
+
+        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+
+17.  Unzip the file:
+
+        unzip awscliv2.zip
+
+18.  See where the current AWS CLI is installed:
+
+        which aws
+
+    It should be `/usr/bin/aws`.
+
+19.  Update it:
+
+        sudo ./aws/install --bin-dir /usr/bin --install-dir /usr/bin/aws-cli --update
+
+20.  Check the version of AWS CLI:
+
+        aws --version
+
+    It should now be updated.
+
+21.  Configure the CLI:
+
+        aws configure
+
+22.  For `AWS Access Key ID`, paste in the access key ID you copied earlier.
+
+23.  For `AWS Secret Access Key`, paste in the secret access key you copied earlier.
+
+24.  For `Default region name`, enter `us-east-1`.
+
+25.  For `Default output format`, enter `json`.
+
+26.  Download `kubectl`:
+
+        curl -o kubectl https://amazon-eks.s3.us-west-2.amazonaws.com/1.16.8/2020-04-16/bin/linux/amd64/kubectl
+
+27.  Apply execute permissions to the binary:
+
+        chmod +x ./kubectl
+
+28.  Copy the binary to a directory in your path:
+
+        mkdir -p $HOME/bin && cp ./kubectl $HOME/bin/kubectl && export PATH=$PATH:$HOME/bin
+
+29.  Ensure `kubectl` is installed:
+
+        kubectl version --short --client
+
+30.  Download `eksctl`:
+
+        curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+
+31.  Move the extracted binary to `/usr/bin`:
+
+        sudo mv /tmp/eksctl /usr/bin
+
+32.  Get the version of `eksctl`:
+
+        eksctl version
+
+33.  See the options with `eksctl`:
+
+        eksctl help
+
+### Provision an EKS Cluster
+
+1.  Provision an EKS cluster with three worker nodes in `us-east-1`:
+
+        eksctl create cluster --name dev --version 1.16 --region us-east-1 --nodegroup-name standard-workers --node-type t3.micro --nodes 3 --nodes-min 1 --nodes-max 4 --managed
```

Comment:
as per instruction on the lab video, this command should be changed to "eksctl create cluster --name dev --region us-east-1 --nodegroup-name standard-workers --node-type t3.micro --nodes 3 --nodes-min 1 --nodes-max 4 --managed"